### PR TITLE
Add auto scroll textbox

### PR DESCRIPTION
**Pull request by EarableFW-Intern**

### CODE REVIEW

File: visual/EB_Bluetooth_temperature_and_battery_level/frmMain.cs [hunk 1]
```diff
@@ -88,6 +89,8 @@ private void SetText(string text)
             else
             {
                 this.tbx_output.Text = text;
+                this.tbx_output.SelectionStart = this.tbx_output.Text.ToString().Length;
+                this.tbx_output.ScrollToCaret();
```
Comment (HocJ2me) [inline visual/EB_Bluetooth_temperature_and_battery_level/frmMain.cs:93]: auto scroll is working good

File: visual/EB_Bluetooth_temperature_and_battery_level/frmMain.cs [hunk 2]
```diff
@@ -104,10 +107,17 @@ private void btn_stop_log_Click(object sender, EventArgs e)
 
         private void tmr_logging_data_Tick(object sender, EventArgs e)
         {
-            Port_deviceTest.WriteLine("AT + TEMPERATURE");
-            Console.WriteLine("AT + TEMPERATURE");
-            Port_deviceTest.WriteLine("AT + BATTERYLEVEL");
-            Console.WriteLine("AT + BATTERYLEVEL");
+            try
```
Comment (HocJ2me) [inline visual/EB_Bluetooth_temperature_and_battery_level/frmMain.cs:110]: add try catch to break blocking but it **not working**

Comment (EarableFW-Intern) [reply visual/EB_Bluetooth_temperature_and_battery_level/frmMain.cs:110]: thankiu alot, i will try to break out the blocking when bluetooth over

**Review by HocJ2me — COMMENTED** [review]
good but not enoght

**Pull request merged**